test-infra/prow/config.yaml

```diff
@@ -127,6 +127,8 @@ periodics:
     - image: gcr.io/compute-image-tools-test/osconfig-tests:latest
       args:
       - "-out_dir=/artifacts"
+      - "-test_project_id=compute-image-osconfigagent"
```

Comment:
If this is a new project there's a bunch of stuff we need to do to set it up, let me know and I can walk you through it. Also we should probably create a pool of projects for osconfig tests to run in as we will have locking issues.

Reply:
yeah, i did some setup around iam. But i was not sure, if that is going to be enough.
Btw, if these are some default steps that we always do when we setup a project for compute-image-test-*, would it be better to just write an automation to do the basic setup?